src/BumpChart.js

```diff
@@ -0,0 +1,69 @@
+import {extent} from "d3-array";
+import {constant} from "d3plus-common";
+
+import {default as Plot} from "./Plot";
+
+/**
+    @class BumpChart
+    @extends Plot
+    @desc Creates a bump chart based on an array of data.
+    @example <caption>the equivalent of calling:</caption>
+new d3plus.Plot()
+  .discrete("x")
+  .shape("Line")
+  .x("x")
+  .y2("y")
+  .yConfig({
+    tickFormat: val => {
+      const data = this._filteredData;
+      const xDomain = extent(data.map(d => d.x));
+      const startData = data.filter(d => d.x === xDomain[0]);
+      return startData.find(d => d.y === val).id;
```

Comment:
Also, we're going to want real labels (and not IDs) for these tick marks. So let's use the internal `drawLabel` function that d3plus-viz makes available:

```jsx
const d = startData.find(d => d.y === val);
return this._drawLabel(d, d.i);
```